lib/mongo/database.rb

```diff
@@ -148,12 +152,14 @@ def collections
     #
     # @return [ Hash ] The result of the command execution.
     def command(operation, opts = {})
+      if opts[:write]
```

Comment:
What's the intention here?  My understanding is that drivers should not make any special effort to include writeConcern in a generic command.